### DISCUSSION TITLE
Load initial entry point as `src=`?

### DISCUSSION THREAD

**Discussion by jkone27** [nojaf/vite-plugin-fable#10] [Ideas]
* https://github.com/vitejs/vite/pull/9981 > seems this was merged? 

<img width="654" alt="image" src="https://github.com/user-attachments/assets/9facc2c0-2b50-47ad-b9af-063952191dd4">

**Comment by nojaf** [comment]
This probably would require some devServer middleware to append that header on F# files.
```
req.headers['sec-fetch-dest'] === 'script'
```

I would accept a PR for this if it turns out to be the case.

**Comment by jkone27** [comment]
i notice with latest versions this just works - https://github.com/jkone27/feliz-vite/blob/main/src/index.html , so maybe we can just update docs?
